src/reducers/listModalReducer.js

```diff
@@ -0,0 +1,12 @@
+const listModalReducer = (listModal = [], action) => {
+  switch (action.type) {
+    case "OPEN_LIST_MODAL":
+      return listModal.concat(action.cards);
+    case "CLOSE_LIST_MODAL":
+      return [];
```

Comment:
It looks like there's a bug where if you open the modal, then close it, then try opening it again, it doesn't open. I think it's because of this line. This should be `return Immutable.List()`